init.d/codedeploy-agent

```diff
@@ -17,11 +17,14 @@
 #                    the deployment artifacts on to this instance.
 ### END INIT INFO
 
+# Source function library.
+. /etc/rc.d/init.d/functions
```

Comment:
This uses systemctl to run start, stop, restart commands. systemctl doesn't stop the agent for the first time when you issue the stop command. This seems to be happening only on RHEL boxes. Could you please address this issue before we could release the changes?
Thanks.